### ISSUE TITLE
abnormal parkinsonian - like behavior

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3432]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

abnormal parkinsonian - like behavior 

>2. Synonyms (e.g., absent spleen)

parkinsonism 

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

the display of analogous human parkinsonian behavior by mice, this behavior may not translate as exact Parkinson's symptoms such as rigidity, akinesia, and tremors

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID: 31500908

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal behavior

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by sbello** [comment]
I think we need to discuss at a uPheno call with how this would fit or if it would fit in a phenotype ontology.

It groups a set of behaviors that may sometimes but not always be lumped as parkinsonism. It is heavily used in papers an may be useful but is problematic.

**Comment by sbello** [comment]
Not going to add this
